src/Internal/ApplyArgs.js

```diff
@@ -9,6 +9,7 @@ if (typeof BROWSER_RUNTIME != "undefined" && BROWSER_RUNTIME) {
   lib = require("@emurgo/cardano-serialization-lib-nodejs");
```

Comment:
What if we made a js file `src/Internal/CSL-wrapped.js` which imports and wraps the needed CSL, and then we import that from all foreign code? Then it's trivial to check all foreign imports use the wrapped one via a grep which searches for the unwrapped one

Reply:
But won't spago move all the js files around during build? Using relative path requires doesn't seem straightforward.